keyboards/nazare160/alpha/rules.mk

```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = STM32F072
```

Comment:
```suggestion
MCU = STM32F072

# Bootloader selection
BOOTLOADER = stm32-dfu
```

Reply:
Committed in [bd52d50](https://github.com/qmk/qmk_firmware/pull/16822/commits/bd52d5069c706a3cc4cf27fe0dcff833774e1176)